diagnostics_tools/include/diagnostic_tools/periodic_event_status.h

```diff
@@ -66,9 +66,26 @@ class PeriodicEventStatusParams final {
 
   const Diagnostic &stale_diagnostic() const { return stale_diagnostic_; }
 
+  PeriodicEventStatusParams &short_term_avg_window(double short_term_avg_window) {
+    short_term_avg_window_ = short_term_avg_window;
+    return *this;
+  }
+
+  double short_term_avg_window() const { return short_term_avg_window_; }
+
+  PeriodicEventStatusParams &long_term_avg_window(double long_term_avg_window)
+  {
```

Comment:
@patapufate nit^2: be consistent with the style (curly brace on the next line vs curly brace on the same line).

Reply:
Ok.